docs/content/components/prompts/yes-or-no-choice.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Yes or no choice
+titleTemplate: ":title · Prompts · Noora · Tuist"
+description: A component that prompts the user to answer a yes or no question.
+---
+
+# Yes or no choice
+
+This component is a simple component that prompts the user to answer a yes or no question. It is a good way to get a quick answer from the user.
```

Comment:
> It is a good way to get a quick answer from the user.

This is the same as for the single choice. I'd either remove it or have a more component-specific description.